### DISCUSSION TITLE
Callout heading font weight

### DISCUSSION THREAD

**Discussion by AleenaSA** [twilio-labs/paste#3073] [Q&A] [answered]
Here are the librayr versions I'm using:
`
        "@twilio-paste/core": "^17.7.0",
        "@twilio-paste/icons": "^9.4.3",`

In a sample application the callout component works as expected. However, in our project repo I get this warning
`Warning: Failed prop type: The prop 'as' is marked as required in 'CalloutHeading', but its value is 'undefined'.`

The warning is resolved if I explicitly provide the 'as' prop, for e.g. `<CalloutHeading as="header">`
 But the font weight of heading is 400 as opposed to the expected 700. 

code:
`
<Callout variant="neutral">
                    <CalloutHeading>
                        You can not update any Organization roles from this
                        page.{' '}
                        <Anchor
                            href='#'
                        >
                            Click here to update Organization roles
                        </Anchor>
                    </CalloutHeading>
                </Callout>`
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/29952750/223346293-9ee25eb6-a987-4b9d-8a8c-d9c4789bdc70.png">

Any pointers on how I can resolve this?

**Comment by SiTaggart** [comment, marked as answer]
Hi @AleenaSA 

Can you provide a Codesandboox reproduction of this for us using the latest version of Paste?

I suspect it's a version issue in your application.

Thanks

**Reply by AleenaSA** [reply]
Hi @SiTaggart, in a code sandbox it works perfectly fine with the same version, I'm getting the error only in our application.

**Reply by SiTaggart** [reply]
OK, well I would still update you paste core and icons versions to the latest anyway as you're a major version behind now.

I'll consider this resolved for now.

**Reply by AleenaSA** [reply]
Hey @SiTaggart I have updated paste to v18, but I'm getting missing props error for components like Box, Anchor:

The following attributes seems to be required
nonce={undefined} onResize={undefined} onResizeCapture={undefined}

**Reply by SiTaggart** [reply]
Hi @AleenaSA 

I noticed you initially linked to a Codesandbox and again, I can't replicate the issues in that environment either.

What I did notice is that there are a lot of dev dependencies listed, each of these might be including things and subsequent dependencies to your apps dependency tree that might cause conflicts. You are going to have to work through resolving your dependency tree correctly in your app. But as you can see, the Codesandbox doesn't show the type issues you describe.
